docs/overview/debugging/index.rst

```diff
@@ -0,0 +1,11 @@
+==============
+Debugging LVGL
```

Comment:
```suggestion
.. _debugging:

==============
Debugging LVGL
```

Reply:
Hi @vwheeler63 
Could you share how you develop the documents?
Is there extension for VSCode? I mainly use this one.
How to build the doc locally to see if everything is fine?

Reply:
> Hi @vwheeler63 Could you share how you develop the documents? Is there extension for VSCode? I mainly use this one. How to build the doc locally to see if everything is fine?

Hi, @XuNeo !

There is about to be a new `./docs/README.md` merged into the `master` branch that contains most of what you will need to know.  Until it is merged you can read it here:

http://crystal-clear-research.com/for_gabor/demo/README.html

And also, because the reStructuredText documentation on the topic of hyperlinks is so jumbled and unclear, after many hours of studying (the reStructuredText documentation at the docutils website) and testing, I wrote this for myself:

http://crystal-clear-research.com/notes_on_reStructuredText/reStructuredText_Vic_Notes.html#sphinx-hyperlinks

Note:  I have been using *exclusively* ***Sphinx Hyperlink Syntax*** and have not found it lacking any capabilities, and it solves the unfortunate maintenance problems presented by ***reStructuredText Hyperlink Syntax***.  While I wrote up a lot of clarifications on ***reStructuredText Hyperlink Syntax*** in that article and tested it all, I strongly recommend NOT using it in LVGL documentation because of the maintenance problems it brings with it.  :-)

Reply:
I have successfully built the doc, thank you for your detailed instructions!